### ISSUE TITLE
Add parameters to Compile-ObjectsInNavContainer.ps1 

### ISSUE THREAD

**Issue by ekeukens** [microsoft/navcontainerhelper#474]
I like the Hack on enlarging filter parameter. But wouldn't it be better to add all parameters of standard compile cmdlet to this cmdlet (and pass them through one on one),

If not, it might be handy to set filter field as output (so you can see what is selected)?

See example.
![AddFilterToOutput](https://user-images.githubusercontent.com/43138953/58570381-2ad6a400-8238-11e9-9a35-298a8b952325.png)

**Comment by freddydk** [comment]
Later versions of the underlying cmdlet actually do support the parameter - but the navcontainerhelper function needs to work with all versions (2018, bc etc.)
I will consider adding some additional output/logging.

**Comment by freddydk** [comment]
Have added this to the function (before the parameter insertion)

```
        if ("$filter" -ne "") {
            Write-Host "Compiling objects with $filter"
        }
        else {
            Write-Host "Compiling all objects"
        }
```
and also a line stating that it is generating symbols (if symbolloading is enabled)

**Comment by freddydk** [comment]
Shipped in 0.6.1.3
